src/app/Food.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
The license header is fine to leave, but we don't need it. I thought the project config turned it off, but you may also need to change it templates some time.